src/builder.rs

```diff
@@ -219,12 +224,14 @@ impl SourceMapBuilder {
             src_col,
             src_id,
             name_id,
+            is_range,
         };
         self.tokens.push(raw);
         raw
     }
 
     /// Adds a new mapping to the builder.
+    #[allow(clippy::too_many_arguments)]
```

Comment:
TBH, clippy is absolutely right to complain here. Trying to review the tests for this on github, I completely lose track of what all these parameters mean (without inlay hints).

Reply:
Yeah, but I'm not sure how should I fix it.

Reply:
Yes, me neither :-( I think this is fine this way, just wanted to call it out as a future improvement opportunity.